source/index.txt

```diff
@@ -16,6 +16,37 @@ introduces key concepts in MongoDB, presents the query language, and
 provides operational and administrative considerations and procedures
 as well as a comprehensive reference section.
 
+.. _getting-started:
+
+Getting Started
+---------------
+
+MongoDB provides a :gettingstarted:`Getting Started Guide </shell>` in the
+following editions.
+
+.. list-table::
+   :class: index-table
+
+   * - :gettingstarted:`mongo Shell Edition </shell>`
+
+       :gettingstarted:`Node.JS Edition </node>`
+
+     - :gettingstarted:`Python Edition </python>`
+
+       :gettingstarted:`C++ Edition </cpp>`
+
+     - :gettingstarted:`Java Edition </java>`
+
+       :gettingstarted:`C# Edition </csharp>`
+
+     -
+
+Topics
+------
+
+The MongoDB manual is under constant development. See the :doc:`/about`
+for more information on the MongoDB Documentation project.
```

Comment:
You might want to get rid of these sentences and clarify that the list of topics here is not exhaustive.